packages/cozy-harvest-lib/src/components/__snapshots__/OAuthForm.spec.js.snap

```diff
@@ -1,38 +1,54 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`OAuthForm should bypass reconnect button when updating an account 1`] = `<React.Fragment />`;
+exports[`OAuthForm should bypass reconnect button when updating an account 1`] = `<div />`;
 
 exports[`OAuthForm should handle oauth cancelation 1`] = `
-<React.Fragment>
-  <withI18n(withClient(withKonnectorLocales(TriggerErrorInfo))
-    className="u-mb-1"
-    error={[Error: OAUTH_CANCELED]}
-    konnector={
-      Object {
-        "slug": "test-konnector",
-      }
-    }
-  />
-  <DefaultButton
-    busy={false}
-    className="u-mt-1"
-    disabled={false}
-    extension="full"
-    label="oauth.connect.label"
-    onClick={[Function]}
-  />
-</React.Fragment>
+<div>
+  <div
+    class="styles__Infos___2ZZVs styles__Infos--danger___3wKYc u-mb-1"
+  >
+    <div
+      class="styles__Stack--m___1tSpV"
+    >
+      <div
+        class="styles__Infos-description___7P1NP"
+      >
+        <div
+          class="styles__Stack--xs___2R5lW"
+        >
+          <h6
+            class="MuiTypography-root u-error MuiTypography-h6 MuiTypography-gutterBottom"
+          >
+            Canceled
+          </h6>
+        </div>
+      </div>
+    </div>
+  </div>
+  <button
+    class="styles__c-btn___-2Vnj styles__c-btn--full___2VmR1 styles__c-btn--center___16_Xh u-mt-1"
+    type="submit"
+  >
+    <span>
+      <span>
+        oauth.connect.label
```

Comment:
It seems that here https://github.com/cozy/cozy-libs/pull/1804/commits/5b7db192cb641a47aa806e04080543722f557674#diff-3eb34f56944764b1f3833c7ab9d70f5c41244b75c5f024d4d0019e2461c35e3eR22 the string is translated, but not for `oauth.connect.label`. Is it a bug?

Reply:
No this is not a bug. The t function is mocked to return the translation key itself. We still have translation in TriggerErrorInfo component which uses I18n from cozy-ui directly.